ci/tasks/send-hg-metric.yml

```diff
@@ -0,0 +1,26 @@
+---
+platform: linux
+
+image_resource:
+  type: registry-image
+  source:
+    repository: alpine
+    tag: 3.12.0
+
+inputs:
+  - name: src
+  - name: release-info
+
+params:
+  METRIC_NAME: metricname
+  METRIC_VALUE: metricvalue
+
+run:
+  path: bash
+  dir: src
+  args:
+    - -eu
+    - |
+        apk add netcat-openbsd
+        echo "Sending metric ((METRIC_NAME))"
+        nc "((HOSTED_GRAPHITE_ACCOUNT_ID)).carbon.hostedgraphite.com" 2003 <<<"((HOSTED_GRAPHITE_API_KEY)).((METRIC_NAME)) ((METRIC_VALUE))"
```

Comment:
Maybe just check this works (I know its pretty straight forward) and then link to the metric from HG in the "How have you tested this" section of the PR description?

Reply:
Added it to the JIRA ticket

Reply:
👍